data/mmissions.txt

```diff
@@ -205,9 +206,8 @@ event "mgf: canaveral naval yard done"
 	planet "Canaveral Naval Yard"
 		landscape land/space0
 		attributes station
-		description `To further advance the Republic Navy and secure humanity's future, the government built a new orbital naval yard to allow production of larger and more advanced ships and remains easily upgradable for any future improvement and expansion. The naval yard is very heavily armed with countless of artillery turrets and advanced anti-missile system spread across the surface. There's also always a fleet carrier in nearby orbit, always within clear visual range.`
-		spaceport `The spaceport have few basic accommodations, mostly as resting area for shipyard workers.`
-		bribe 1.9
+		description `To further advance the Republic Navy and secure humanity's future, the government built a new orbital naval yard to allow production of larger and more advanced ships. It is also designed to be easily upgradable for any future improvement and expansion. The naval yard is very heavily armed with countless numbers of artillery turrets and advanced anti-missile systems spread across the surface. There's also always a fleet carrier in nearby orbit, always within clear visual range.`
+		spaceport `The spaceport has a few basic accommodations, mostly resting areas for the shipyard workers.`
```

Comment:
It makes it so that they will make you pay to try to land, a.k.a. you have to pay almost twice your net worth.
No landing on this station if the Republic is hostile.

Reply:
Except for that is no longer in here?